otherlibs/win32unix/stat.c

```diff
@@ -117,23 +117,13 @@ static value stat_aux(int use_64, __int64 st_ino, struct _stat64 *buf)
 
 static int convert_time(FILETIME* time, __time64_t* result, __time64_t def)
 {
-  SYSTEMTIME sys;
-  FILETIME local;
+  unsigned __int64 utime = *((unsigned __int64*)time);
```

Comment:
It does indeed - it goes to show why you shouldn't try to code when you're supposed to be in bed :-/ It's not totally clear to me from the documentation whether this is an Itanium problem (so definitely irrelevant) or an x86-64 problem but, either way, this is hardly a performance-critical piece of code!